runtime/moonbase/src/lib.rs

```diff
@@ -1252,7 +1261,26 @@ impl Default for AssetType {
 
 impl From<MultiLocation> for AssetType {
```

Comment:
How does `change_existing_asset_type()` relate to this, or are they entirely unrelated?

Reply:
Unrelated. I added `change_existing_asset_type` so that in the future we can change the old statemine prefix to the new one for outgoing xcm to Statemine, but we will not use it for now